invenio_accounts/models.py

```diff
@@ -463,3 +468,153 @@ def delete_by_user(cls, method, user):
         """Unlink a user from an external id."""
         with db.session.begin_nested():
             cls.query.filter_by(id_user=user.id, method=method).delete()
+
+
+class DomainOrg(db.Model):
+    """Domain organisation."""
+
+    __tablename__ = "accounts_domain_org"
+
+    id = db.Column(db.Integer(), primary_key=True, autoincrement=True)
+
+    pid = db.Column(db.String(255), unique=True, nullable=True)
+    """Persistent identifier for organisation."""
+
+    name = db.Column(db.String(255), nullable=False)
+    """Name of organisation."""
+
+    json = db.Column(
```

Comment:
will be used on zenodo to store e.g. ``country_code``, `in_eu``, ``in_cern_ms``, ``in_cern_ams`` which can be used for statistics purposes.